server/rest/src/main/java/org/infinispan/rest/ResponseWriter.java

```diff
@@ -49,7 +50,19 @@ void writeResponse(ChannelHandlerContext ctx, FullHttpRequest request, NettyRest
          if (entity instanceof byte[]) {
             responseContent.writeBytes((byte[]) entity);
          } else if (entity instanceof ByteArrayOutputStream) {
-            responseContent.writeBytes(((ByteArrayOutputStream)entity).toByteArray());
+            responseContent.writeBytes(((ByteArrayOutputStream) entity).toByteArray());
+         } else if (entity instanceof Throwable) {
+            Throwable t = Util.getRootCause((Throwable) entity);
+            StringBuilder sb = new StringBuilder();
+            if (t.getSuppressed().length > 0) {
+               for(Throwable s : t.getSuppressed()) {
+                  sb.append(s.getMessage());
+                  sb.append(System.lineSeparator());
```

Comment:
IMO a HTTP response shouldn't depend on the what OS the server is running on, so it would be better to use `\n` instead.